x-pack/plugins/apm/server/lib/traces/get_top_traces.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment';
+import {
+  TRANSACTION_NAME,
+  TRANSACTION_ID,
+  TRANSACTION_DURATION
+} from '../../../common/constants';
+import { get, sortBy } from 'lodash';
+
+export async function getTopTraces({ setup }) {
+  const { start, end, esFilterQuery, client, config } = setup;
+
+  const duration = moment.duration(end - start);
+  const minutes = duration.asMinutes();
+
+  const params = {
+    index: config.get('apm_oss.transactionIndices'),
+    body: {
+      size: 0,
+      query: {
+        bool: {
+          must: {
+            exists: {
+              field: 'trace.id' // todo: move to constant? or TS?
```

Comment:
I thought we could get rid of the constants.js but for these cases I think the constants still make sense. 
I was actually going to make a test that checks the constants against our typescript interfaces, so we are sure they don't get out of sync.

so tldr: unless you know a way to do this with typescript, use a constant.